brainstormforce/ultimate-addons-for-gutenberg
# Fix: Social share - Editor gap between icons should matches the frontend.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/e3b8a9ed-7c78-43e6-84a4-7677106697ed?redirect=true&view=06c01065-22db-4b44-9a83-f0eceb06e66a&t=attributes

### Screenshots

See here - https://share.getcloudapp.com/Wnuyq4RW

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop Social Share.
- Check editor matches the frontend or not.
- Check by adjusting the gap between the Icons setting. It should work the same as the editor and frontend.

### Checklist:
